strings/ion_laws.json

```diff
@@ -535,10 +521,10 @@
         "PRESS B",
         "SMELL LIKE THE MAN YOUR MAN COULD SMELL LIKE",
         "PIRATE VIDEO GAMES",
-        "WATCH PORNOGRAPHY",
         "INSULT THE LIZARDS",
         "FLIRT WITH THE LIZARDS",
-        "GAS THE LIZARDS"
```

Comment:
I personally feel we should promote this type of behavior :thinking:

Reply:
Slaughter, Sacrifice, Maim... sure.

Gas, is a bit too... Final Solution for me.